sklearn/utils/_param_validation.py

```diff
@@ -154,6 +155,10 @@ def validate_params(parameter_constraints):
         Note that the *args and **kwargs parameters are not validated and must not be
         present in the parameter_constraints dictionary.
 
+    skip_nested_validation : bool, default=True
```

Comment:
Is the intended default `False`?

Reply:
The intended default is True, I made a mistake in the signature.

The reason I chose True as default is because this is almost always what we want. However if we chose to always pass it explicitely, I don't care if we chose True or False as default, or even not set default at all.